src/main/resources/com/puppycrawl/tools/checkstyle/grammar/java.g

```diff
@@ -121,7 +121,7 @@ tokens {
     PATTERN_VARIABLE_DEF; RECORD_DEF; LITERAL_record="record";
     RECORD_COMPONENTS; RECORD_COMPONENT_DEF; COMPACT_CTOR_DEF;
     TEXT_BLOCK_LITERAL_BEGIN; TEXT_BLOCK_CONTENT;
-    TEXT_BLOCK_LITERAL_END;
+    TEXT_BLOCK_LITERAL_END; LITERAL_yield="yield"; SWITCH_RULE;
```

Comment:
please move them both to new line, we try to keep  them groupped only if related.

Reply:
Fixed.